cpp/src/arrow/csv/converter.cc

```diff
@@ -56,6 +57,26 @@ inline bool IsWhitespace(uint8_t c) {
   return c == ' ' || c == '\t';
 }
 
+// Updates data and size to not include leading/trailing whitespace
+// characters.
+inline void TrimWhiteSpace(const uint8_t*& data, uint32_t& size) {
```

Comment:
According to the style guide, we should use `const uint8_t** data, uint32_t* size` here :-)

Reply:
Nice catch.  Slightly embarrassing. Fixed.